tests/contracts/equal.clar

```diff
@@ -110,6 +110,10 @@
     (ok (is-eq (some 0x01) (some 0x02)))
 )
 
+(define-public (call-optional-some-and-err-unequal)
```

Comment:
```suggestion
(define-public (call-optional-some-and-none-unequal)
```

Reply:
fixed in [67d812d](https://github.com/stacks-network/clarity-wasm/pull/174/commits/67d812d41cd70e7489c4a050bd1b84d5ba98aaca)